scheduler/pkg/agent/rproxy_grpc.go

```diff
@@ -206,16 +208,8 @@ func (rp *reverseGRPCProxy) setTrailer(ctx context.Context, trailer metadata.MD,
 	}
 }
 
-func (rp *reverseGRPCProxy) ModelInfer(ctx context.Context, r *v2.ModelInferRequest) (*v2.ModelInferResponse, error) {
-	logger := rp.logger.WithField("func", "ModelInfer")
-	internalModelName, externalModelName, err := rp.extractModelNamesFromContext(ctx)
-	if err != nil {
-		return nil, err
-	}
-	r.ModelName = internalModelName
-	r.ModelVersion = ""
-
-	// to sync between scalingMetricsSetup and scalingMetricsTearDown calls running in go routines
+// to sync between scalingMetricsSetup and scalingMetricsTearDown calls running in go routines
+func (rp *reverseGRPCProxy) syncScalingMetrics(internalModelName string, logger log.FieldLogger) {
```

Comment:
`reverseGRPCProxy` has already access to a logger, there is no need to pass it here?